src/encoded/tests/test_structured_data.py

```diff
@@ -19,6 +17,23 @@
 SAME_AS_EXPECTED_REFS = {}
 SAME_AS_NOREFS = {}
 
+# Ideally this flag would be False, i.e. we would like to use the actual/real (live)
+# schemas that are defined in (the schemas directory of) this repo for testing; but at
+# least currently (2024-01-10) these are undergoing a lot of change and leading to frequent
+# and annoying test breakage; so setting this to True will cause these tests to use a dump of
+# the schemas which were previously saved into a static file (data/test-files/schemas_dump.json).
+USE_SAVED_SCHEMAS_RATHER_THAN_LIVE_SCHEMAS = True
```

Comment:
I think this is fine but could we at least read the file from dcicutils (it is in one of David's PRs as well) so we don't have this huge dump file replicated across repos? Referencing the exact same file could come with advantages as well so we ensure there is no divergence while still actively developing.

Reply:
David added these changes, so will discuss with him.